### DISCUSSION TITLE
Feature-Request: Index-Shifting

### DISCUSSION THREAD

**Discussion by FRauANtjeK** [LiMinggang/madedit-mod#323] [Ideas]
Hi,

I would love to have "index-shifting"
That means, that you shift the index from 0000 to another value. 

Example: Would be great for C64 files that uasually start at 0801. So you do not have to calculate every jump-adress :)

There may be other computers too, that start binary programs not at 0000 but at another adress...

Thank you in advance,

FF

**Comment by LiMinggang** [comment]
Are you talking about the "Offset" below?
![image](https://user-images.githubusercontent.com/4847132/144954992-9d60d3d5-2480-4c24-bf7c-0170ae5f8e7c.png)

**Comment by FRauANtjeK** [comment]
Yes, exactly.

**Reply by LiMinggang** [reply]
I've no idea where to put the setting item. Any suggestion?

**Comment by FRauANtjeK** [comment]
maybe:

alt-1 text-mode
alt-2 column-mode  (i dont know the english word - in german "Spalten Modus"
alt-3 hex-mode
alt-4 offset-mode

brgds,


frank


Am Di., 7. Dez. 2021 um 14:51 Uhr schrieb Minggang Li <
***@***.***>:

**Reply by LiMinggang** [reply]
Not sure if a new mode was needed, it's still the same as Hex mode anyway. What I just asked is that I need a way to config the offset start value.
